src/features/contactmoment/ContactmomentNotitie.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <utrecht-heading model-value :level="2">Notitieblok</utrecht-heading>
+    <textarea
+      ref="notitieRef"
+      @change="notitieChanged"
+      v-model="contactmomentStore.notitie"
+      v-focus
+      v-bind="$attrs"
+    >
+    </textarea>
+  </div>
+</template>
```

Comment:
het notitieblok verpest de hoogte van de pagina een beetje. volgens mij helpt het als we #app ` display: flex; flex-direction: column;` geven, en main `flex: 1`;

Reply:
Dat lost het probleem helaas niet op. gezien verwachte komende wijzigingen wilde ik hier nu niet teveel tijd in steken

Reply:
besproken